app/.gitignore

```diff
@@ -1 +1,2 @@
 /build
+/keystore.jks
```

Comment:
Just curious - why does this need to be added to .gitignore?

Reply:
I placed the keystore files in the project because I had trouble setting a relative path.

Reply:
Can you please remove from the PR if this is temporary?

Reply:
Sure thing. Do you have a specific path for the `keystore.jks` that you'd suggest? I can include the path in the `keystore.properties` file.

Reply:
Path to `keystore.jks` should be somewhere unversioned on your own computer. `keystore.properties` file goes in the root project folder (you can see it's currently in the `.gitignore` there, so it's not versioned either), and then you can put the absolute path to the `keystore.jks` file in that file, like:

```
store=D:/Dropbox/MUSER/keystore/keystore.jks
...
```

Reply:
Great. Thank you for letting me know.